bleak/backends/bluezdbus/device.py

```diff
@@ -0,0 +1,12 @@
+from bleak.backends.device import BLEDevice
+
+
+class BLEDeviceBlueZDBus(BLEDevice):
```

Comment:
Ooof. This seems like going in the wrong direction. I know it is not the case right now, but ideally `BleakScanner` and `BleakClient` should be the only classes that have a platform-specific implementation. With some effort, `BleDevice` and the service/characteristic/descriptor classes could be made universal. We don't have to do that now, but let's not make more work just to be undone later.

Reply:
Oops, sorry. And fully agreed. I wanted to make things more orthogonal between backend implementations, but your suggestion is much better. 

OTOH: I'm not sure I've introduced "more work": now all classes are treated the same way for all implementations. So that basically means that all classes could get a `backend_private` property where the backend could store whatever bits they wanted and if they wanted.

Hmm, I see an analogy to your idea in the other thread about making `BleakClient` backend-independent...